### ISSUE TITLE
[AccessibilityMutator] Add MDFTextAccessibilityOptions as an option to all of our mutators.

### ISSUE THREAD

**Issue by mohammadcazig** [material-components/material-components-ios#3078]
If we are supporting dark and light mode. We need to give users the option to give MDFTextAccessibilityOptions as an input to decide for light or dark text options instead of always going with our default which is light.

---
#### Internal data
- Associated internal bug: [b/117178977](http://b/117178977)

**Comment by romoore** [comment]
This is not required for the accessibility level 4 project.

**Comment by romoore** [comment]
This is still not required for the accessibility level 4 project.

**Comment by romoore** [comment]
Closing this due to inactivity greater than 6 months. Please feel free to reopen if you believe it should be worked on in the near future.
